src/rewrite_clj/node/whitespace.clj

```diff
@@ -57,10 +71,18 @@
   "Create whitespace node."
   [s]
   {:pre [(string? s)
-         (re-matches #"(\s|,)+" s)
-         (not (re-matches #".*[\n\r].*" s))]}
+         (re-matches #"\s+" s)
+         (not (re-matches #".*[\n\r,].*" s))]}
```

Comment:
The `,` in the 3rd guard isn't needed because it can't pass the 2nd guard anyway. I think we still need the 3rd guard tho.